### PR TITLE
bs4 enrollment history

### PR DESCRIPTION
**Pull request by scaustin34**
# PR Checklist

Please check if your PR fulfills the following requirements
- [ ] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [ ] Any endpoint modified in the PR only responds to the expected MIME types.
- [ ] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [x] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/187141970

# A brief description of the changes

Current behavior:

New behavior:

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/helpers/insured/families_helper.rb [hunk 1]
```diff
@@ -263,7 +263,7 @@ def enrollment_state_label(enrollment)
     label = group[condition] || { text: enrollment.aasm_state.to_s.titleize, color: 'grey' }
     # Coverage reinstated is a special case where the aasm state is something else but we want to show it as reinstated in "green" (active) scenarios
     label = state_groups[:coverage_reinstated][:default] if enrollment.is_reinstated_enrollment? && label[:color] == 'green'
-    content_tag(:span, label[:text], class: "label label-#{label[:color]}")
```
Comment (charlienparker) [inline app/helpers/insured/families_helper.rb:266]: should the px-2 class be behind the feature flag?

File: app/javascript/src/enrollment.js [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
-};
 
-// For when family home page loaded through clicking off of the families index page
-if (enrollmentToggleCheckbox != null || enrollmentToggleCheckbox != undefined) {
-    enrollmentToggleCheckbox.addEventListener('click', toggleDisplayEnrollments);
-};
+        if (enrollmentToggleCheckbox.checked) {
+            for (let i = 0; i < initiallyHiddenEnrollmentPanels.length; i++) {
+                initiallyHiddenEnrollmentPanels[i].classList.remove("hidden");
+                enrollmentToggleButton.innerText = l10n("");
```
Comment (charlienparker) [inline app/javascript/src/enrollment.js:18]: does the empty string need to be localized? or is there a missing string we need here?

File: app/javascript/src/enrollment.js [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+            }
+        } else {
+            for (let i = 0; i < initiallyHiddenEnrollmentPanels.length; i++) {
+                initiallyHiddenEnrollmentPanels[i].classList.add("hidden");
+                enrollmentToggleButton.innerText = "Show All Enrollments";
```
Comment (charlienparker) [inline app/javascript/src/enrollment.js:23]: should this be localized? I know I've seen issues trying to call l10n in our js packs, so maybe localizing this is out of scope. we should probably work towards finding a solution for that though

Comment (scaustin34) [reply app/javascript/src/enrollment.js:23]: yeah, wasnt able to get the translations working. Do we have any way of doing this that we dont know about @kristinmerbach ?

File: app/helpers/insured/families_helper.rb (hunk 1 above)
Comment (scaustin34) [reply app/helpers/insured/families_helper.rb:266]: probably, but this is all going to get reworked with the enrollment tile ticket - this change was just to make thing a little more functional until then and it barely changes the exisitng/non-bs UI

File: app/javascript/src/enrollment.js (hunk 3 above)
Comment (kristinmerbach) [reply app/javascript/src/enrollment.js:23]: I would add the translation to the toggle button as a data attribute, then in the js, grab that
